OrbitBase/include/OrbitBase/Logging.h

```diff
@@ -73,13 +80,13 @@
 
 // Internal.
 #if defined(_WIN32)
-#define PLATFORM_LOG(message)       \
+#define PLATFORM_PRINT(message)     \
   do {                              \
     fprintf(stderr, "%s", message); \
     OutputDebugStringA(message);    \
   } while (0)
 #else
-#define PLATFORM_LOG(message)       \
+#define PLATFORM_PRINT(message)     \
   do {                              \
```

Comment:
For just one line that already has to be terminated with `;` the do-while is not necessary.

Reply:
Done.